### PR TITLE
Animate home stack border radius

### LINKED ISSUE #16283: Animate home stack border radius
### Summary

Animate home stack border radius, so corners don't look square when opening and closing home-stack

Originally requested here: https://github.com/status-im/status-mobile/pull/15893#issuecomment-1548046435

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes: https://github.com/status-im/status-mobile/issues/16283

status: ready

### CODE REVIEW

File: src/js/worklets/shell/home_stack.js [hunk 1]
```diff
@@ -21,51 +21,43 @@ export function stackZIndex (stackId, selectedStackId) {
...
 // Derived values for home stack (container)
-export function homeStackOpacity (homeStackState) {
+export function homeStackOpacity(homeStackState) {
   return useDerivedValue(
     function () {
       'worklet'
       switch (homeStackState.value) {
       case constants.OPEN_WITH_ANIMATION:
 	return withTiming(1, constants.LINEAR_EASING);
-	break;
```
Comment (Parveshdhull) [inline src/js/worklets/shell/home_stack.js:31]: removing redundant break statement

**Review by njiddasalifu — COMMENTED** [review]
I see some test checks failing, I suggest try the E2E test to make it pass. Just a suggestion

File: src/utils/worklets/shell.cljs [hunk 2]
```diff
@@ -59,6 +59,10 @@
   [home-stack-state-sv top-home-stack-position]
   (.homeStackTop ^js home-stack-worklets home-stack-state-sv top-home-stack-position))
 
+(defn home-stack-border-radius
+  [home-stack-state-sv]
```
Comment (J-Son89) [inline src/utils/worklets/shell.cljs]: what is `sv`? why not just `home-stack-state` since that's what you have on js side.

Comment (Parveshdhull) [reply src/utils/worklets/shell.cljs]: https://github.com/status-im/status-mobile/pull/15247#discussion_r1125021144

**Review by J-Son89 — APPROVED** [review]

**Comment by Parveshdhull** [comment]
hi @status-im/mobile-qa, skipping manual testing, as PR just introduces small animation change. Please can you confirm if failed e2e tests are not related to PR?

File: src/utils/worklets/shell.cljs (hunk 2 above)
Comment (Parveshdhull) [reply src/utils/worklets/shell.cljs]: changed, thank you for suggestion

**Comment by pavloburykh** [comment]
> hi @status-im/mobile-qa, skipping manual testing, as PR just introduces small animation change. Please can you confirm if failed e2e tests are not related to PR?

@Parveshdhull thanx for the PR. I will check e2e failures and provide you with feedback. We might need to re-run some of the failed e2e.

**Comment by Parveshdhull** [comment]
> > hi @status-im/mobile-qa, skipping manual testing, as PR just introduces small animation change. Please can you confirm if failed e2e tests are not related to PR?
> 
> @Parveshdhull thanx for the PR. I will check e2e failures and provide you with feedback. We might need to re-run some of the failed e2e.

yeah np, take your time. Thank you for picking it

**Comment by pavloburykh** [comment]
> yeah np, take your time. Thank you for picking it

@Parveshdhull I have checked and looks like there is no need to re-run e2e as failures are not PR related. So you are good to merge it.

**Pull request merged**